app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -1911,6 +1911,309 @@ class DatabaseMigrationAndroidTest {
+
+    db_v36.query("SELECT * FROM $tableName").use {
+      it.moveToNext()
+      assertThat(it.columnCount).isEqualTo(12)
+      assertThat(it.count).isEqualTo(1)
+      assertThat(it.string("uuid")).isEqualTo(mhUuid)
+    }
+  }
+
+  @Test
+  fun migrate_patient_address_from_35_to_36() {
+    val db_v35 = helper.createDatabase(version = 35)
+
+    val addressTableName = "PatientAddress"
+    val patientTable = "Patient"
+    val addressUuid = "uuid"
+
+    db_v35.execSQL("""
+      INSERT INTO $addressTableName VALUES(
+        '$addressUuid',
+        'colony or village',
+        'district',
+        'state',
+        'country',
+        'created-at',
+        'updated-at',
+        NULL,
+        'recorded-at')
+    """)
+
+    db_v35.execSQL("""
+      INSERT INTO $patientTable VALUES(
+        'patientUuid',
+        '$addressUuid',
+        'patient-name',
+        'name',
+        'MALE',
+        NULL,
+        'ACTIVE',
+        'created-at',
+        'updated-at',
+        'null',
+        'IN_FLIGHT',
+        'recorded-at',
+        25,
+        'age-updated-at',
+        'dob');
+    """)
+
+    val db_v36 = helper.migrateTo(version = 36)
+
+    db_v36.query("SELECT * FROM $addressTableName").use {
+      it.moveToNext()
+      assertThat(it.columnCount).isEqualTo(8)
+      assertThat(it.count).isEqualTo(1)
+      assertThat(it.string("uuid")).isEqualTo(addressUuid)
+    }
+
+    db_v36.query("SELECT * FROM $patientTable").use {
+      it.moveToNext()
+      assertThat(it.count).isEqualTo(1)
+      assertThat(it.string("addressUuid")).isEqualTo(addressUuid)
+      assertThat(it.string("uuid")).isEqualTo("patientUuid")
+    }
+  }
+
+  @Test
+  fun migrate_phone_number_from_35_to_36() {
+    val db_v35 = helper.createDatabase(version = 35)
+
+    val phoneNumberTableName = "PatientPhoneNumber"
+    val patientTable = "Patient"
+    val patientUuid = "patientUuid"
+    val phoneUuid = "phoneUuid"
+
+    db_v35.execSQL("""
+      INSERT INTO "PatientAddress" VALUES(
+        'addressUuid',
+        'colony or village',
+        'district',
+        'state',
+        'country',
+        'created-at',
+        'updated-at',
+        NULL,
+        'recorded-at')
+    """)
+
+    db_v35.execSQL("""
+      INSERT INTO $patientTable VALUES(
+        '$patientUuid',
+        'addressUuid',
+        'patient-name',
+        'name',
+        'MALE',
+        NULL,
+        'ACTIVE',
+        'created-at',
+        'updated-at',
+        'null',
+        'IN_FLIGHT',
+        'recorded-at',
+        25,
+        'age-updated-at',
+        'dob');
+    """)
+
+    db_v35.execSQL("""
+      INSERT INTO $phoneNumberTableName VALUES(
+      '$phoneUuid',
+      '$patientUuid',
+      '981615191',
+      'mobile',
+      0,
+      'created-at',
+      'updated-at',
+      'null',
+      'recorded-at')
+    """)
+
+    val db_v36 = helper.migrateTo(version = 36)
+
+    db_v36.query("SELECT * FROM $patientTable").use {
```

Comment:
Is this verification necessary? Since there is no foreign key from patient to phone number, there's no reason for this verification, right?

Reply:
Not really. I just added this test to verify that `Patient` table is unchanged due to this migration.

Reply:
I will remove this.